iscript/src/iscript/constants.py

```diff
@@ -4,11 +4,10 @@
 MAC_PRODUCT_CONFIG = {
     "firefox": {
         "designated_requirements": (
-            """=designated => ( """
-            """(anchor apple generic and certificate leaf[field.1.2.840.113635.100.6.1.9] ) """
-            """or (anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] """
+            """=designated => """
+            """anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] """
             """and certificate leaf[field.1.2.840.113635.100.6.1.13] and certificate """
-            """leaf[subject.OU] = "%(subject_ou)s"))"""
+            """leaf[subject.OU] = "%(subject_ou)s" """
```

Comment:
Thank you for dealing with this. I know you haven't put this up for review just yet, and maybe you were planning to do this anyways, but I'd like to ask that we document this, and maybe reference a bug (or something) that will help us when we need to make adjustments to this in the future. It's been a black box ever since we began doing mac signing :(.